java/1.2/modules/ROOT/pages/configure-java-module-acb-edit.adoc

```diff
@@ -0,0 +1,331 @@
++
+This is the target variable assigned to the *New* operation.
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+[[configuring-invoke-static]]
+=== Configuring the Invoke Static Operation
+
+Invoke static methods with the arguments you provide. 
+
+To configure the attributes for *Invoke Static* in the Anypoint Code Builder UI:
+
+. Click the *Invoke Static* component on the canvas.
+. In the *General* tab:
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+Only the static methods from the class are displayed.
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+=== Configuring the Validate Type Operation
+
+Use this operation to validate that the instance is an `instanceof` the expected class.
+
+. Click the *Validate type* component on the canvas.
+. In the *General* tab:
+.. Select the *Class* the instance belongs to.
+.. Enter the name of the class *Instance* to validate.
+.. Select *Accept subtypes* to accept subtypes of a class. 
++
+Unselect this option if the class instance has to be of the exact same class. 
+
+== Example
+
+[[creating-java-class]]
+=== Creating a Java Class
+
+Create Java classes containing methods to use in your application flows. The classes you create in your project are not available to other projects in your Anypoint Code Builder workspace.
+
+. In your Anypoint Code Builder project, create a new Java package called `com.me` under `src/main/java`.
+. Create a new Java class called `TaxCalculator.java`, and add the implementation, for example:
++
+[source,java,linenums]
+----
+package com.me;
+
+public class TaxCalculator {
+
+    private Double percentBaseTax = 7.0;
+
+    public Double calculateTax(Double price, Integer percentAdditionalTax) {
+        return price * (this.percentBaseTax + percentAdditionalTax) / 100;
+    }
+
+    public Boolean isTaxFree(Double price) {
+        if (price < 10) {
+            return true;
+        }
+        return false;
+    }
+}
+---- 
++
+In this example code, there are two methods:
+* `calculateTax`, which receives the `price` and `percentAdditionalTax` parameters.
+* `isTaxFree`, which receives the `price` parameter.
+
+=== Configure an HTTP Trigger
+
+. Click the *Listener* component on the canvas UI.
+. (Optional) Click the edit icon to change the name of the *Listener* component. 
```

Comment:
Kim's guidance says to avoid.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-optionalsteps